TOML.sublime-syntax

```diff
@@ -6,12 +6,14 @@ name: TOML
 file_extensions:
   - toml
   - tml
+  - Cargo.lock
+  - Gopkg.lock
```

Comment:
(note: in case anyone questions this, other syntaxes also specify full filenames in the `file_extensions` section and both Sublime and Syntect support this feature. e.g. see https://github.com/zyxar/Sublime-CMakeLists/blob/ff9a800a4ca942edd095de553ca05fba03b02275/CMake.sublime-syntax#L7)